lib/elixir/lib/map.ex

```diff
@@ -709,10 +709,10 @@ defmodule Map do
   end
 
   @doc """
-  Removes the value associated with `key` in `map` and returns the value
-  and the updated map, or it raises if `key` is not present.
+  Removes and returns the value associated with `key` in `map` alongside
+  the updated map, or raises if `key` is not present.
```

Comment:
I changed this sentence because it should be remain small but kept the rest. :)

Reply:
```suggestion
  the updated map, or raises a `KeyError` exception  if `key` is not present.
```
Ok, but "raises **what** if"? I guess there should be something or that part can be skipped since in line 715 it is mentioned that this function raises error when key is not present

Reply:
"raises" in Elixir is always about exceptions. The details of which exception can be found in the next paragraph. :)